include/fem-dem/cfd_dem_coupling.h

```diff
@@ -287,10 +330,13 @@ class CFDDEMSolver : public GLSVANSSolver<dim>
   PeriodicBoundariesManipulator<dim> periodic_boundaries_object;
   Tensor<1, dim>                     periodic_offset;
   bool                               has_periodic_boundaries;
+  bool                               particle_displaced_in_pbc;
+
+  AdaptiveSparseContacts<dim> sparse_contacts_object;
+  bool                        has_sparse_contacts;
+  unsigned int                contact_build_counter;
 
-  DisableContacts<dim> disable_contacts_object;
-  bool                 has_disabled_contacts;
-  unsigned int         contact_build_number;
+  unsigned int contact_build_number;
```

Comment:
Parameter comments if possible.

Reply:
I added comment, but the document in Doxygen format should be done for the whole file at once in another PR